### PR TITLE
Fixed thing

### PR DESCRIPTION
**Pull request by raghubetina**
Hey team, please give me feedback on...

### CODE REVIEW

**Comment by lenglander** [comment]
Looks great!

**Comment by pmckernin** [comment]
This looks great! Maybe some spacing on whatever.

**Comment by pmckernin** [comment]
LGTM! 🚢

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Puzzles
 
+Fixed whatever
```
Comment (pmckernin) [inline README.md:3]: This looks fixed

**Pull request merged**